react-prebuild.jsx

```diff
@@ -0,0 +1,25 @@
+// @flow
+import React from 'react';
+import {renderToString} from 'react-dom/server';
+import Logo from './kuma/javascript/src/Logo';
+
+let stdinAccumulator = '';
+
+process.stdin.on('data', input => {
+    stdinAccumulator += input.toString();
+});
+
+process.stdin.on('end', () => {
+    const locales = JSON.parse(stdinAccumulator);
+
+    const result = Object.keys(locales).reduce((acc, locale) => ({
+        [locale]: {
+            Logo: renderToString(<Logo
```

Comment:
I think I'd be inclined to not use JSX in this file, and use React.createComponent() (or whatever the function is called) that the JSX compiles too.